homeassistant/helpers/check_config.py

```diff
@@ -12,9 +12,9 @@
 
 from homeassistant import loader
 from homeassistant.config import (  # type: ignore[attr-defined]
-    CONF_CORE,
     CONF_PACKAGES,
     CORE_CONFIG_SCHEMA,
+    HA_DOMAIN,
```

Comment:
This is a re-import. It should use the source and not the alias from a re-import.